programs/ssl/ssl_client2.c

```diff
@@ -1534,13 +1534,18 @@ int main( int argc, char *argv[] )
             {
                 sig_alg_list[i++] = MBEDTLS_TLS13_SIG_ECDSA_SECP521R1_SHA512;
             }
+            else if( strcmp( q, "rsa_pkcs1_sha256" ) == 0 )
+            {
+                sig_alg_list[i++] = MBEDTLS_TLS13_SIG_RSA_PKCS1_SHA1;
+            }
             else
             {
                 mbedtls_printf( "unknown signature algorithm %s\n", q );
                 mbedtls_printf( "supported signature algorithms: " );
                 mbedtls_printf( "ecdsa_secp256r1_sha256 " );
                 mbedtls_printf( "ecdsa_secp384r1_sha384 " );
                 mbedtls_printf( "ecdsa_secp521r1_sha512 " );
+                mbedtls_printf( "rsa_pkcs1_sha256 " );
```

Comment:
ditto

Reply:
I Just change back to align with the exist coding style as comments https://github.com/ARMmbed/mbedtls/pull/5127#discussion_r755114957.  Please help check whether we can close this conversation.